src/main/app/routes/start.ts

```diff
@@ -9,5 +9,5 @@ export default express.Router()
     res.render('claim/start')
   })
   .post(Paths.main, (req, res) => {
```

Comment:
please use req: express.Request, res: express.Response by default for req and res